Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/moth.yml

```diff
@@ -46,14 +46,76 @@
         maxvol: 6
         reagents:
         - ReagentId: Nutriment
-          Quantity: 2
-        - ReagentId: Vitamin
           Quantity: 4
+        - ReagentId: Vitamin
+          Quantity: 2
+
+# base for our special sauces
+- type: entity
+  parent: BaseItem
+  id: BaseFoodMothSauce
+  name: sauce bowl
+  abstract: true
+  description: A small bowl for condiment. Not reusable.
+  components:
+  - type: Drink
+    solution: food
+    refillable: false
+    isOpen: true
+  - type: DrawableSolution
+    solution: food
+  - type: InjectableSolution
+    solution: food
+  - type: DrainableSolution
+    solution: food
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 15
+  - type: SolutionTransfer
+    maxTransferAmount: 15
+  - type: Sprite
+    netsync: false
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: condimentbowl
+  - type: Icon
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: condimentbowl
+  - type: TrashOnEmpty
+    solution: food
```

Comment:
Well done, but the trash bit makes me wonder... What if we required a smol bowl or another smaller vessel in the recipes for this and then changed the trash item to that same vessel? Things like steaks used to work like that, pies and spaget still does. We'd only need to change the base entity for the trash part, but it would still mean changing all recipes that use this base because there's no parent recipes... so far.

Reply:
Had it to be like that when I used the soup as a base, it kinda worked but to have the entity itself go from "Soup" to "Empty Bowl" I had to try to drink from it, only then it would be like "oh ye im empty" and become a normal bowl again.
Could potentially re add that if the trash part is really an issue.

Reply:
Oh, I see, yeah, that feels kind of hacky and unreliable, just have it be a single use thing. Don't want to risk someone losing bowls to this.